.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -3,6 +3,22 @@ name: Bug report
 about: Create a bug report to help us improve
 ---
 
+### Issue workflow progress
+
+<!-- PLEASE DO NOT REMOVE THIS SECTION -->
+
+_Progress of the issue based on the [Contributor Workflow](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md#a-typical-contributor-workflow)_
+
+- [ ] 1. The issue provides a reproduction available on GitHub, Stackblitz or CodeSandbox
+> Make sure to fork this template and run `yarn generate` in the terminal.
+>
+> Please make sure the Codegen and plugins version under `package.json` matches yours.
```

Comment:
`graphql-eslint` does not depend on codegen and its plugins, the reproduction template should be different